integration_test/EmitVerilog/lint.mlir

```diff
@@ -6,6 +6,7 @@
 // RUN: verilator --lint-only --top-module TESTSIMPLE %t1.sv
```

Comment:
Lint checks are great, but I was referring to the register-like behavior. You'd have to add an always@ block and some toggling behavior, then check that the register is flipping in the test.

Reply:
Oh... right. I'll add an actual test along those lines.